generated_api_shadow/envoy/config/core/v3/protocol.proto

```diff
@@ -71,6 +71,24 @@ message UpstreamHttpProtocolOptions {
   bool auto_san_validation = 2;
 }
 
+message AlternateProtocolsCacheOptions {
```

Comment:
(This comment not specific to your PR, just curiosity.) Are the protos in `generated_api_shadow` automatically generated? If yes do you know why they're in the repository as opposed to generated at build time?

Reply:
Oh, I missed this one. They're generated by running:

./tools/proto_format/proto_format.sh fix

I don't understand why that doesn't happen at build time, but it is SUPER slow so that's probably part of it :/